docker/Dockerfile

```diff
@@ -18,4 +18,4 @@ RUN npm install -g flowise
 
 WORKDIR /data
 
-CMD "flowise"
+ENTRYPOINT ["npx", "flowise", "start"]
```

Comment:
Thanks for this change, I'm using this image in K8S and I'm executing a different command because the one in the current image doesn't work.

But I don't think it's necessary to use `npx` given that Flowise is being installed globally in the previous step (`-g` flag). It should be more straightforward and faster to run `flowise start`. This is the command we are executing.